pulsar-io/debezium/postgres/pom.xml

```diff
@@ -44,6 +44,13 @@
       <version>${debezium.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>42.2.25</version>
```

Comment:
why not use variable instead of hard code?

Reply:
see the description

> Note: the version is not shared on purpose because we should leave the driver dependencies separated since the two connectors are used in different ways. (For example, when we'll upgrade Debezium to 1.8.x we'll need to remove the override and keep the 42.3.x version)

I can add a local variable actually